src/lib/dataEntities/networks.ts

```diff
@@ -329,9 +349,100 @@ export const l2Networks: L2Networks = {
   },
 }
 
-const getNetwork = async (
+const isParentChain = (chain: Chain): chain is ParentChain => {
+  return chain && 'partnerChainIDs' in chain
+}
+const isChildChain = (chain: Chain): chain is ChildChain => {
+  return chain && 'partnerChainID' in chain
+}
+
+const isL1Chain = (chain: Chain): chain is L1Network => {
+  return chain && isParentChain(chain) && !isChildChain(chain)
+}
+
+const isL2Chain = (chain: Chain): chain is L2Network => {
+  return chain && isChildChain(chain) && !isOrbitChain(chain)
+}
+
+const isOrbitChain = (chain: Chain): chain is OrbitChain => {
+  return chain && 'isOrbit' in chain && chain.isOrbit
+}
+
+export const getL1Chains = () => {
+  return Object.entries(Networks).reduce((acc, [key, value]) => {
+    if (isL1Chain(value)) {
+      acc[key] = value
+    }
+    return acc
+  }, {} as L1Networks)
+}
+
+export const getL2Chains = () => {
+  return Object.entries(Networks).reduce((acc, [key, value]) => {
+    if (isL2Chain(value)) {
+      acc[key] = value
+    }
+    return acc
+  }, {} as L2Networks)
+}
+
+export const getParentChains = () => {
+  return Object.entries(Networks).reduce((acc, [key, value]) => {
+    if (isParentChain(value)) {
+      acc[key] = value
+    }
+    return acc
+  }, {} as ParentChains)
+}
+
+export const getChildChains = () => {
+  return Object.entries(Networks).reduce((acc, [key, value]) => {
+    if (isChildChain(value)) {
+      acc[key] = value
+    }
+    return acc
+  }, {} as ChildChains)
+}
+
+export const getOrbitChains = () => {
+  return Object.entries(Networks).reduce((acc, [key, value]) => {
+    if (isOrbitChain(value)) {
+      acc[key] = value
+    }
+    return acc
+  }, {} as OrbitChains)
+}
+
+export const getParentForNetwork = (chain: Chain) => {
```

Comment:
we're calling them Chains sometimes and Networks sometimes
do we want to use one name across everything?